kstatemachine-coroutines/src/commonMain/kotlin/ru/nsk/kstatemachine/CoroutinesStateMachine.kt

```diff
@@ -22,6 +22,7 @@ suspend fun createStateMachine(
     autoDestroyOnStatesReuse: Boolean = true,
     enableUndo: Boolean = false,
     doNotThrowOnMultipleTransitionsMatch: Boolean = false,
+    metaInfo: StateMetaInfo? = null,
```

Comment:
looks this should be in a builder block, same as for transitions
see `BuildingStateMachine`

Reply:
is available in `BuildingStateMachine` through IState